test/index.test.js

```diff
@@ -279,6 +280,24 @@ describe('Segment.io', function() {
         analytics.assert(object.context);
         analytics.assert(!object.context.amp);
       });
+
+      it('should add a list of enabled integrations when `addEnabledMetadata` is set', function() {
+        var opts = extend({ addEnabledMetadata: true }, options);
+        var Analytics = analytics.constructor;
+        var ajs = new Analytics();
+        ajs.use(Segment);
+        ajs.use(integration('other'));
+        ajs.initialize({ 'Segment.io': opts, other: {} });
+
+        segment = ajs._integrations['Segment.io'];
```

Comment:
You should initialize this integration as done up on/around L40 so you don't have to reach into this private member